libs/blocks/section-metadata/section-metadata.js

```diff
@@ -55,6 +55,12 @@ function handleDelay(time, section) {
   setTimeout(() => { section.classList.remove('hide-sticky-section'); }, getDelayTime(time));
 }
 
+function handleAnchor(anchor, section) {
+  if (!(anchor || section)) return;
```

Comment:
One last nit: `anchor` is a string, right? If the string is empty, the conditional returns `false` and it ends up adding an empty ID. Wouldn't it be better to have it as `!anchor || !section` or `!(anchor && section)`?

Reply:
you're correct, I made the changes.